src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -226,66 +212,29 @@ class CompanyStep extends React.Component {
-                                errorText={this.getErrorText('addressStreet')}
-                            />
-                            <Text style={[styles.mutedTextLabel, styles.mt1]}>{this.props.translate('common.noPO')}</Text>
-                            <View style={[styles.flexRow, styles.mt4]}>
-                                <View style={[styles.flex2, styles.mr2]}>
-                                    <ExpensiTextInput
-                                        label={this.props.translate('common.city')}
-                                        onChangeText={value => this.clearErrorAndSetValue('addressCity', value)}
-                                        value={this.state.addressCity}
-                                        errorText={this.getErrorText('addressCity')}
-                                    />
-                                </View>
-                                <View style={[styles.flex1]}>
-                                    <StatePicker
-                                        onChange={value => this.clearErrorAndSetValue('addressState', value)}
-                                        value={this.state.addressState}
-                                        hasError={this.getErrors().addressState}
-                                    />
-                                </View>
-                            </View>
-                            <ExpensiTextInput
-                                label={this.props.translate('common.zip')}
-                                containerStyles={[styles.mt4]}
-                                keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
-                                onChangeText={value => this.clearErrorAndSetValue('addressZipCode', value)}
-                                value={this.state.addressZipCode}
-                                errorText={this.getErrorText('addressZipCode')}
-                            />
-                        </>
-                    )}
-
+                    <AddressForm
+                        values={{
+                            street: this.state.addressStreet,
+                            city: this.state.addressCity,
+                            zipCode: this.state.addressZipCode,
+                            state: this.state.addressState,
+                        }}
+                        errors={{
+                            street: this.getErrors().addressStreet,
+                            city: this.getErrors().addressCity,
+                            zipCode: this.getErrors().addressZipCode,
+                            state: this.getErrors().addressState,
+                        }}
+                        onFieldChange={(values) => {
+                            const renamedValues = {};
+                            _.each(values, (value, inputKey) => {
+                                const renamedInputKey = lodashGet(this.renamedFields, inputKey, inputKey);
```

Comment:
NAB, if we are not using `this.renamedFields` again we could just declare them in this method. Only thought to mention it because we are doing that in the `RequestorStep` and could keep it consistent.